src/Components/Pages/Results/Results.tsx

```diff
@@ -71,6 +71,33 @@ export const Results: React.FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const openAccordionsPrint = () => {
```

Comment:
this is a really smart way to handle opening everything for the print! I did not think about doing it like this.

Reply:
I was very jazzed that it worked too!